### ISSUE TITLE
Getting an error after worker inactivity

### ISSUE THREAD

**Issue by bergundy** [temporalio/sdk-core#57]
Happens when running the `sdk-node` worker.

```
[Error: Poll response from server was malformed: PollWorkflowTaskQueueResponse { task_token: [], workflow_execution: None, workflow_type: None, previous_started_event_id: 0, started_event_id: 0, attempt: 0, backlog_count_hint: 0, history: None, next_page_token: [], query: None, workflow_execution_task_queue: None, scheduled_time: None, started_time: None, queries: {} }]
```

**Comment by Sushisource** [comment]
Note for myself: This happens when the long poll times out and gets an empty response. We just need to ignore it and retry.

**Comment by Sushisource** [comment]
This was fixed back in #78
